### ISSUE TITLE
Docker compose failing at mongodb step on Arch Linux

### ISSUE THREAD

**Issue by berquist** [nitish6174/cclib-web#7]
From `sudo docker-compose build`:

```
Step 5/18 : RUN service mongodb start
 ---> Running in a7e2b3f72c20
 * Starting database mongodb
   ...fail!
ERROR: Service 'app' failed to build: The command '/bin/sh -c service mongodb start' returned a non-zero code: 1
```

Is this because I already have mongodb installed and running? I can test on an Ubuntu 12.04 box.

**Comment by nitish6174** [comment]
Having mongodb installed shouldn't be the reason I believe as I too had mongodb installed and running on my system.

I am using ubuntu 16.04. Not sure if this is a platform-specific issue.

**Comment by berquist** [comment]
Yes, I thought the whole point of using Docker was that it's an isolated container. I'll try it on my 12.04 install. I also have a Mac but have heard bad things about installing/running Docker on it.

**Comment by ghutchis** [comment]
@berquist - I've run docker on a Mac. It's not "bad" - you just want to make sure you get the official Docker Mac build:
https://docs.docker.com/docker-for-mac/
